### ISSUE TITLE
Prisma scan fails for deck v1.22.0 with security vulnerability for go v1.20.4

### ISSUE THREAD

**Issue by Niranjan-K** [Kong/deck#949]
Prisma scan failed with security vulnerability for deck version 1.22.0 (current latest version). It says the go version used to build the deck is golang version 1.20.4 (attached screenshot). but as per the release notes of v1.22.0, it states that the golang has been updated to 1.20.5.

![Screenshot 2023-06-19 at 1 46 16 pm](https://github.com/Kong/deck/assets/5525926/3a8f4b24-a4f9-4a54-8fa7-9ca21e2404e8)

**Comment by Niranjan-K** [comment]
Noticed that in the git actions (workflow), it is picking go version as 1.20.4. Can this be the reason why it is being built using golang 1.20.4 even thou it is mentioned in the Dockerfile as 1.20.5 (https://github.com/Kong/deck/blob/v1.22.0/Dockerfile#L1)?

![Screenshot 2023-06-19 at 2 02 57 pm](https://github.com/Kong/deck/assets/5525926/1a44d004-f2ad-49a5-aaaf-c2e408384fcf)

**Comment by GGabriele** [comment]
Hi @Niranjan-K ,

We just released [v1.22.1](https://github.com/Kong/deck/releases/tag/v1.22.1) which is now using Go v1.20.5:

```
$ strings /opt/homebrew/bin/deck | grep 'go1\.'
go1.20.5
vendor/gol
```
